src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -0,0 +1,628 @@
+     border-radius: 1000px 1000px 0 0;
+   }
+   .full .rs-bar,
+   .half .rs-bar {
+     width: 50%;
+   }
+   .half.left .rs-bar,
+   .half.right .rs-bar,
+   .quarter .rs-bar {
+     width: 100%;
+   }
+   .full .rs-bar,
+   .half.left .rs-bar,
+   .half.right .rs-bar {
+     top: 50%;
+   }
+   .bottom .rs-bar {
+     top: 0;
+   }
+   .half.right .rs-bar,
+   .quarter.right .rs-bar {
+     right: 100%;
+   }
+
+   .rs-handle.rs-move {
+     cursor: move;
+   }
+   .rs-readonly .rs-handle.rs-move {
+     cursor: default;
+   }
+   .rs-path {
+     display: block;
+     height: 100%;
+     width: 100%;
+   }
+   .rs-split .rs-path {
+     -webkit-border-radius: 1000px 1000px 0 0;
+     border-radius: 1000px 1000px 0 0;
+     overflow: hidden;
+     height: 50%;
+     position: absolute;
+     top: 0;
+     z-index: 2;
+   }
+
+   /*** tooltip styles ***/
+   .rs-tooltip {
+     position: absolute;
+     cursor: default;
+     border: 1px solid transparent;
+     z-index: 10;
+     color: var(--primary-text-color);
+   }
+   .full .rs-tooltip {
+     top: 45%;
+     left: 50%;
+   }
+   .bottom .rs-tooltip {
+     top: 0;
+   }
+   .top .rs-tooltip {
+     bottom: 0;
+   }
+   .right .rs-tooltip {
+     left: 0;
+   }
+   .left .rs-tooltip {
+     right: 0;
+   }
+   .half.top .rs-tooltip,
+   .half.bottom .rs-tooltip {
+     left: 50%;
+   }
+   .half.left .rs-tooltip,
+   .half.right .rs-tooltip {
+     top: 50%;
+   }
+   .rs-tooltip .rs-input {
+     outline: 0 none;
+     border: none;
+     background: transparent;
+   }
+   .rs-tooltip-text {
+     font-size: ${Math.round(this.clientWidth / 6)}px;
+     border-radius: 7px;
+     text-align: center;
+   }
+   .rs-tooltip.edit,
+   .rs-tooltip .rs-input {
+     padding: 5px 8px;
+   }
+   .rs-tooltip.hover,
+   .rs-tooltip.edit:hover {
+     border: 1px solid #aaaaaa;
+     cursor: pointer;
+   }
+   .rs-readonly .rs-tooltip.edit:hover {
+     border-color: transparent;
+     cursor: default;
+   }
+
+   /*** handle types ***/
+   .rs-handle {
+     border-radius: 1000px;
+     outline: 0 none;
+     float: left;
+   }
+   .rs-handle.rs-handle-square {
+     border-radius: 0px;
+   }
+   .rs-handle-dot {
+     border: 1px solid #aaaaaa;
+     padding: 6px;
+   }
+   .rs-handle-dot:after {
+     display: block;
+     content: "";
+     border: 1px solid #aaaaaa;
+     height: 100%;
+     width: 100%;
+     border-radius: 1000px;
+   }
+
+   /*** theming - colors ***/
+   .rs-seperator {
+     border: 1px solid #aaaaaa;
+   }
+   .rs-border {
+     border: 1px solid #aaaaaa;
+   }
+   .rs-path-color {
+     background-color: #ffffff;
+   }
+   .rs-range-color {
+     background-color: #54bbe0;
+   }
+   .rs-bg-color {
+     background-color: #ffffff;
+   }
+   .rs-handle {
+     background-color: #838383;
+   }
+   .rs-handle-dot {
+     background-color: #ffffff;
+   }
+   .rs-handle-dot:after {
+     background-color: #838383;
+   }
+
+    </style>
+    `;
+  }
+
+  private tooltip(e) {
```

Comment:
Maybe we keep tooltip out of first version to keep it simpler?

Reply:
Tooltip shows what you have set the temperature to

Reply:
ok